src/script/interpreter.cpp

```diff
@@ -1417,182 +1393,7 @@ bool EvalScript(vector<vector<unsigned char> >& stack, const CScript& script, un
-                                    const CTxOut newLockOutput = checker.GetOutputOffsetFromCurrent(1);
-                                    if (!newLockOutput.nValue.IsExplicit())
-                                        return set_error(serror, SCRIPT_ERR_WITHDRAW_VERIFY_BLINDED_AMOUNTS);
-                                    if (newLockOutput.IsNull() || newLockOutput.scriptPubKey != relockScript || newLockOutput.nValue.GetAmount() < lockValueRequired)
-                                        return set_error(serror, SCRIPT_ERR_WITHDRAW_VERIFY_RELOCK_SCRIPTVAL);
-                                }
-
-                                const CTxOut withdrawOutput = checker.GetOutputOffsetFromCurrent(0);
-                                if (!withdrawOutput.nValue.IsExplicit())
-                                    return set_error(serror, SCRIPT_ERR_WITHDRAW_VERIFY_BLINDED_AMOUNTS);
-                                if (withdrawOutput.nValue.GetAmount() < withdrawVal)
-                                    return set_error(serror, SCRIPT_ERR_WITHDRAW_VERIFY_OUTPUT_VAL);
-
-                                CScript expectedWithdrawScriptPubKey;
-                                if (vcontract[0] == 'P' && vcontract[1] == '2' && vcontract[2] == 'S' && vcontract[3] == 'H')
-                                    expectedWithdrawScriptPubKey = CScript() << OP_HASH160 << std::vector<unsigned char>(vcontract.begin() + 4, vcontract.begin() + 24) << OP_EQUAL;
-                                else if (vcontract[0] == 'P' && vcontract[1] == '2' && vcontract[2] == 'P' && vcontract[3] == 'H')
-                                    expectedWithdrawScriptPubKey = CScript() << OP_DUP << OP_HASH160 << std::vector<unsigned char>(vcontract.begin() + 4, vcontract.begin() + 24) << OP_EQUALVERIFY << OP_CHECKSIG;
-                                else
-                                    return set_error(serror, SCRIPT_ERR_WITHDRAW_VERIFY_FORMAT);
-
-                                if (withdrawOutput.scriptPubKey != expectedWithdrawScriptPubKey)
-                                    return set_error(serror, SCRIPT_ERR_WITHDRAW_VERIFY_OUTPUT_SCRIPT);
-
-#ifndef BITCOIN_SCRIPT_NO_CALLRPC
-                                if (GetBoolArg("-validatepegin", DEFAULT_VALIDATE_PEGIN) && !checker.IsConfirmedBitcoinBlock(merkleBlock.header.GetHash(), flags & SCRIPT_VERIFY_INCREASE_CONFIRMATIONS_REQUIRED, GetArg("-peginconfirmationdepth", DEFAULT_PEGIN_CONFIRMATION_DEPTH)))
-                                    return set_error(serror, SCRIPT_ERR_WITHDRAW_VERIFY_BLOCKCONFIRMED);
-#endif
-                            } catch (std::exception& e) {
-                                // Probably invalid encoding of something which was deserialized
-                                return set_error(serror, SCRIPT_ERR_WITHDRAW_VERIFY_FORMAT);
-                            }
-                        }
-                    } // else...OP_NOP3
-                }
-                break;
-
-                default:
```

Comment:
+1

Reply:
what does +1 mean here

Reply:
oh, just that I appreciate this code being removed..

Reply:
+1 from me too (for this whole file's diff)

Reply:
my pleasure.